createdisk.sh

```diff
@@ -24,6 +24,11 @@ ${SSH} core@${VM_IP} -- 'sudo find /var/log/ -iname "*.log" -exec rm -f {} \;'
 # Remove moby-engine package
 ${SSH} core@${VM_IP} -- 'sudo rpm-ostree override remove moby-engine'
 
+# Add binfmt for using other platform
+${SSH} core@${VM_IP} 'sudo bash -x -s' <<EOF
+  rpm-ostree install qemu-user-binfmt qemu-user-static-aarch64 qemu-user-static-x86_64
+EOF
```

Comment:
Could be just `sudo rpm-ostree install qemu-user-binfmt qemu-user-static-aarch64 qemu-user-static-x86_64` as the line above instead adding `bash -x -s`.
Do we want to make this conditional on `ARCH` to only install the correct package? Dunno how often one will want to run aarch64 images on x86_64.

Reply:
It would be ideal to only include x86_64 on an Aarch64 image, and vice versa. Though the size does not increase that much